### ISSUE TITLE
Volume cannot be set with buttons from 6% up

### ISSUE THREAD

**Issue by Ayuus** [aunefyren/bluesound_alt#4]
This issue is present in the official integration, too, and also reported there, but maybe you can fix this?
For Bluesound devices only, volume cannot be set from 6% to 7% and up via volume buttons. 
Slider works OK.
Refer to https://community.home-assistant.io/t/volume-control-problem-through-media-player-bluesound/354549

**Comment by aunefyren** [comment]
Huh, what an odd problem. Thank you for reporting the issue, I am definitely interested in this being fixed. I am currently migrating the integration to the new HA media standard so it could be that this resolves the issue as well.

**Comment by Ayuus** [comment]
Yes, very strange. I am not the only one with this issue, but not everyone seems to have trouble with it. Appearantly, works OK for you:-)

**Comment by Ayuus** [comment]
Any news on this?

**Comment by aunefyren** [comment]
@Ayuus, can you check if the issue persists in v1.0.0?

**Comment by Ayuus** [comment]
No, sorry, I use HA native integration, which suits my needs.

**Comment by aunefyren** [comment]
Alrigth, no worries. Closing this then.
